### DISCUSSION TITLE
Support for Canon CR3 raw format

### DISCUSSION THREAD

**Discussion by Gnuclear** [sylikc/jpegview#184] [Ideas]
Currently, JPEGView is able to open Canon's CR2 raw files, but not the newer CR3 compressed raw files.

**Comment by sylikc** [comment]
Yeah, this is a known issue.  I need some help with moving away from the ancient dcraw to libRAW which would support CR3.

This is the first mentioned issue in #142

**Comment by sylikc** [comment]
merged with PR https://github.com/sylikc/jpegview/pull/211 ... available in 1.3.46
